exercises/complex-numbers/complex_numbers_test.rb

```diff
@@ -0,0 +1,207 @@
+    #skip
+    expected = ComplexNumber.new([4, 6])
+    assert_equal expected, ComplexNumber.new([1, 2]) + ComplexNumber.new([3, 4])
+  end
+
+  def test_subtract_purely_real_numbers
+    #skip
+    expected = ComplexNumber.new([-1, 0])
+    assert_equal expected, ComplexNumber.new([1, 0]) - ComplexNumber.new([2, 0])
+  end
+
+  def test_subtract_purely_imaginary_numbers
+    #skip
+    expected = ComplexNumber.new([0, -1])
+    assert_equal expected, ComplexNumber.new([0, 1]) - ComplexNumber.new([0, 2])
+  end
+
+  def test_subtract_numbers_with_real_and_imaginary_part
+    #skip
+    expected = ComplexNumber.new([-2, -2])
+    assert_equal expected, ComplexNumber.new([1, 2]) - ComplexNumber.new([3, 4])
+  end
+
+  def test_multiply_purely_real_numbers
+    #skip
+    expected = ComplexNumber.new([2, 0])
+    assert_equal expected, ComplexNumber.new([1, 0]) * ComplexNumber.new([2, 0])
+  end
+
+  def test_multiply_purely_imaginary_numbers
+    #skip
+    expected = ComplexNumber.new([-2, 0])
+    assert_equal expected, ComplexNumber.new([0, 1]) * ComplexNumber.new([0, 2])
+  end
+
+  def test_multiply_numbers_with_real_and_imaginary_part
+    #skip
+    expected = ComplexNumber.new([-5, 10])
+    assert_equal expected, ComplexNumber.new([1, 2]) * ComplexNumber.new([3, 4])
+  end
+
+  def test_divide_purely_real_numbers
+    #skip
+    expected = ComplexNumber.new([0.5, 0])
+    assert_equal expected, ComplexNumber.new([1, 0]) / ComplexNumber.new([2, 0])
+  end
+
+  def test_divide_purely_imaginary_numbers
+    #skip
+    expected = ComplexNumber.new([0.5, 0])
+    assert_equal expected, ComplexNumber.new([0, 1]) / ComplexNumber.new([0, 2])
+  end
+
+  def test_divide_numbers_with_real_and_imaginary_part
+    #skip
+    expected = ComplexNumber.new([0.44, 0.08])
+    assert_equal expected, ComplexNumber.new([1, 2]) / ComplexNumber.new([3, 4])
+  end
+
+  def test_absolute_value_of_a_positive_purely_real_number
+    #skip
+    expected = ComplexNumber.new([5])
+    assert_equal expected, ComplexNumber.new([5, 0]).abs
+  end
+
+  def test_absolute_value_of_a_negative_purely_real_number
+    #skip
+    expected = ComplexNumber.new([5])
+    assert_equal expected, ComplexNumber.new([-5, 0]).abs
+  end
+
+  def test_absolute_value_of_a_purely_imaginary_number_with_positive_imaginary_part
+    #skip
+    expected = ComplexNumber.new([5])
+    assert_equal expected, ComplexNumber.new([0, 5]).abs
+  end
+
+  def test_absolute_value_of_a_purely_imaginary_number_with_negative_imaginary_part
+    #skip
+    expected = ComplexNumber.new([5])
+    assert_equal expected, ComplexNumber.new([0, -5]).abs
+  end
+
+  def test_absolute_value_of_a_number_with_real_and_imaginary_part
+    #skip
+    expected = ComplexNumber.new([5])
+    assert_equal expected, ComplexNumber.new([3, 4]).abs
+  end
+
+  def test_conjugate_a_purely_real_number
+    #skip
+    expected = ComplexNumber.new([5, 0])
+    assert_equal expected, ComplexNumber.new([5, 0]).conjugate
```

Comment:
I've got no idea what `conjugate` means.
I think the operations need to be defined in the readme.

Reply:
I think you are right. However, this concerns problems-specifications readme. And any change made there should be propagated to all the tracks that implement this problem.

Maybe automatic generation of READMEs based on problem-specific and track-specific templates would be a good idea?